br/pkg/lightning/restore/table_restore.go

```diff
@@ -406,6 +406,11 @@ func (tr *TableRestore) restoreEngine(
 		if err != nil {
 			return closedEngine, errors.Trace(err)
 		}
+		if rc.status != nil && rc.status.backend == config.BackendTiDB {
+			for _, chunk := range cp.Chunks {
+				rc.status.FinishedFileSize.Add(chunk.Chunk.EndOffset)
```

Comment:
I am not sure TiDB backend would run here(`cp.Status >= checkpoints.CheckpointStatusAllWritten`)😵‍💫?

Reply:
I found there's a saving time for tidb backend

https://github.com/pingcap/tidb/blob/65585a5b3cd1dc13c75a8f2aa1b7d87a1303434e/br/pkg/lightning/restore/table_restore.go#L622-L623